src/Microsoft.ML.AutoML/API/AutoCatalog.cs

```diff
@@ -278,5 +280,14 @@ public RankingExperiment CreateRankingExperiment(RankingExperimentSettings exper
             UserInputValidationUtil.ValidateInferColumnsArgs(path);
             return ColumnInferenceApi.InferColumns(_context, path, labelColumnIndex, hasHeader, separatorChar, allowQuoting, allowSparse, trimWhitespace, groupColumns);
         }
+
+        /// <summary>
+        /// Create a sweepable estimator with a custom factory and search space.
+        /// </summary>
+        internal SweepableEstimator CreateSweepableEstimator<T>(Func<MLContext, T, IEstimator<ITransformer>> factory, SearchSpace<T> ss = null)
```

Comment:
This is internal, but you are creating a sample with it. We want our samples to be created only with public methods so that anyone who wants to use the samples can.

Reply:
KK, so should I temporarily remove that titanic sample until we make it public?

Reply:
Thats what I would do. Since if a user tried to use the sample it wouldn't work for them. Just make sure we add it back in when we do make them public.

Reply:
Sure will do

Reply:
### Resolved